content/MELT/people.md

```diff
@@ -5,19 +5,21 @@ url: /people
 
 We are:
 
-* [Eric Van Wyk](http://www-users.cs.umn.edu/~evw/), Associate Professor
+* [Eric Van Wyk](http://www-users.cs.umn.edu/~evw/), Professor
```

Comment:
Are congratulations in order here?  :-)

Reply:
No - just means I can never be promoted again...